log/internal/encoders/config.go

```diff
@@ -0,0 +1,80 @@
+package encoders
```

Comment:
This file is copied from https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/tree/lib/log/internal/encoders